### DISCUSSION TITLE
v-bind in css

### DISCUSSION THREAD

**Discussion by zzzgit** [vuejs/rfcs#605] [General]
Using v-bind in css, volar says:

<img width="730" alt="image" src="https://github.com/vuejs/rfcs/assets/1060733/ea77ad93-f8fd-4892-a75f-a03448b419b1">

**Comment by posva** [comment]
Remember to use the [Discord chat](https://chat.vuejs.org/) to ask questions or Discussions (if enabled)!

**Comment by ShGKme** [comment]
If you use not a property name but a JS expression, you need to wrap it in a quote:
```css
width: v-bind('style.asideWidth')
```

See documentation: https://vuejs.org/api/sfc-css-features.html#v-bind-in-css

> supports JavaScript expressions (must be wrapped in quotes)

---

But I'd recommend using plain properties in `v-bind` in css to keep CSS variable name simple in the result.
